src/shape/bounding-sphere.js

```diff
@@ -99,10 +100,28 @@ pc.extend(pc, function () {
                 point.copy(ray.direction).scale(t).add(ray.origin);
 
             return true;
+        },
+
+        /**
+         * @function
+         * @name pc.BoundingSphere#intersectsBoundingSphere
```

Comment:
Perhaps another method would be useful to find out if a sphere is actually contains another sphere (contains, means fully inside and not colliding with the edges.